packages/e2e-tests/kit-node/__tests__/kit.spec.ts

```diff
@@ -285,7 +285,8 @@ describe('kit-node', () => {
 						'/^svelte\\//', // serialized with toString
 						'e2e-test-dep-svelte-api-only',
 						'svelte-i18n',
-						'esm-env',
+						'esm-env', // first added by svelte-kit
+						'esm-env', // second added by vite-plugin-svelte
```

Comment:
can we add a check to avoid the duplication or does it not cause any trouble w vite?